### ISSUE TITLE
[NEW] Database Fixups Review UI / CLI

### ISSUE THREAD

**Issue by jhunt** [shieldproject/shield#587]
**Is your feature request related to a problem? Please describe.**

SHIELD has a concept of a "database fixup" - a bit of code that should run once to "fix up" database state caused by previous, buggier versions of SHIELD.  Currently, there is no way to review what fixups exist, when they were run (if ever) and what happened.

**Describe the solution you'd like**
I would like a new **Administration** area, for SHIELD site operators to use, that provides the following, for each database fixup:

 1. Its name
 2. A description of what the fixup is supposed to do (I believe these are stored in-db as markdown)
 3. When the fixup was applied (if ever)
 4. A log of what happened when that fixup ran
 5. A "(re-)run" button, for triggering or re-triggering fixups.

**Comment by jhunt** [comment]
Database support for this is in-place, we just need an API and CLI/Web UI implementations.

**Comment by jhunt** [comment]
Web UI:

<img width="1041" alt="Screen Shot 2019-09-05 at 11 51 40 PM" src="https://user-images.githubusercontent.com/20047/64399891-23454900-d038-11e9-80e0-976c6eb28d32.png">

CLI:

```
→  ./shield -c localdev fixups
ID                                 Name                                                      Created at                Applied at
==                                 ====                                                      ==========                ==========
purge-task-516                     Re-schedule Failed Purge Tasks                            2019-05-14 11:54:00-0400  2019-09-05 23:12:04-0400
agent-status-task-tenant-uuid-522  Associate Orphaned agent-status Tasks with Global Tenant  2019-05-15 08:32:00-0400  2019-09-05 23:13:58-0400
keep-n-460                         Job Keep-N=0 Fix                                          2019-05-15 11:43:00-0400  2019-09-05 23:12:04-0400
```

**Comment by jhunt** [comment]
To get more detail in the CLI:

```
→  ./shield -c localdev fixup keep-n-460
ID:         keep-n-460
Name:       Job Keep-N=0 Fix
Created at: 2019-05-15 11:43:00-0400
Applied at: 2019-09-05 23:12:04-0400
Summary:    The holistic `/v2/tenants/systems/...` API handlers were
            incorrectly skipping the calculation and population of the number
            of kept backups, leading to front-end display of jobs created via
            the Web UI (only) that claim to be keeping "0 backups / X days".

            This has been fixed as of 8.2.0, and this data fixup
            re-calculates the "keep-n" attribute of all jobs that were
            affected.

            See [issue
            #460](https://github.com/shieldproject/shield/issues/460) in
            GitHub for details.
```
